src/js/components/Search.js

```diff
@@ -350,9 +350,11 @@ export default class Search extends Component {
     if (this.props.className) {
       classes.push(this.props.className);
     }
+    if (this.props.iconAlign) {
+      classes.push(CLASS_ROOT + "--icon-align-" + this.props.iconAlign);
```

Comment:
I'm trying to get the existing components to use string templates. Going one by one will take us some time. So instead we are trying to update as we go. 

Would you be kind enough to update Search component to use string templates and classnames?

See the anchor component for an example: https://github.com/grommet/grommet/blob/master/src/js/components/Anchor.js.